storage/sql_migrations/004_statuslist.sql

```diff
@@ -0,0 +1,43 @@
+-- migrate:up
+
+-- status_list_credentials: keeps track of all status list credentials issued by an issuer, and the highest status list index issued for each credential.
+create table status_list_credentials
+(
+    -- status_list_credential: url where this StatusList2021Credential can be downloaded.
+    status_list_credential varchar(500) not null primary key,
+    -- issuer: the DID of the issuer of this statusListCredential.
+    issuer varchar(500) not null,
+    -- page: the n-th StatusList2021Credential issued by this issuer.
+    --      Used to find the credential statusList2021Entries are being issued to. Could require an index in the future.
+    page integer not null,
+    -- last_issued_index: the highest status_list_index issued for this page. Should not be incremented above the max statusListIndex.
+    last_issued_index integer not null
+    -- Ties status list credentials to DID management.
+    --!! There should be a single table for all managed DIDs, not per DID method.
```

Comment:
added. but now the test DIDs need to be written to `vdr_didweb` table to prevent foreign key constraint errors

Reply:
The issuer now produces errors because of this. I'm assuming we are going to have issues like this more often, what would be a good way to deal with this without turning everything into an integration test?

Reply:
When testing a SQL store implementation, it only makes sense to build it as an integration test i.m.o, otherwise you need to mock the SQL/GORM driver which overcomplicates things.

From tests point of view, we could load some default data into it when creating a "test" storage engine?

Reply:
added function `storage.AddDIDtoSQLDB` for now.